### PR TITLE
New parameters for SMBClientConfiguration

### PR DESCRIPTION
**Pull request by Xelu86**
# PR Summary

New parameters have been added to SMBClientConfiguration for WS25. Update related cmdlets to conform to style guide.

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: docset/winserver2025-ps/smbshare/Set-SmbClientConfiguration.md [hunk 1]
```diff
@@ -637,6 +760,54 @@ Accept pipeline input: False
...
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Smb2DialectMin
+
+This parameter specifies the minimum version of the SMB protocol to be used. Acceptable values are:
+
+- None – There is no maximum protocol version specified, the client can use any supported version.
+- SMB202 – SMB 2.0.2 is the maximum version accepted by the SMB client.
```
Comment (nedpyle) [inline docset/winserver2025-ps/smbshare/Set-SmbClientConfiguration.md:792]: these all should say minimum not maximum

File: docset/winserver2025-ps/smbshare/Set-SmbClientConfiguration.md [hunk 2]
```diff
@@ -637,6 +760,54 @@ Accept pipeline input: False
...
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Smb2DialectMin
+
+This parameter specifies the minimum version of the SMB protocol to be used. Acceptable values are:
+
+- None – There is no maximum protocol version specified, the client can use any supported version.
```
Comment (nedpyle) [inline docset/winserver2025-ps/smbshare/Set-SmbClientConfiguration.md:791]: minimum

**Review by nedpyle — CHANGES_REQUESTED** [review]
I will get back to you on a few of these

File: docset/winserver2025-ps/smbshare/Set-SmbClientConfiguration.md [hunk 3]
```diff
@@ -89,11 +89,79 @@ Accept pipeline input: False
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AuditServerDoesNotSupportEncryption
+
+Specifies whether to audit when a server doesn't support encryption. If this parameter is
```
[the remark this answers is not in the document]
Comment (robinharwood) [reply docset/winserver2025-ps/smbshare/Set-SmbClientConfiguration.md:110]: @nedpyle - as discussed

[the remark this answers is not in the document]
Comment (nedpyle) [reply docset/winserver2025-ps/smbshare/Set-SmbClientConfiguration.md:110]: It's good, no further change needed

**Pull request merged**